main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+var (
+	command string
+	resFile string
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		printUsage()
+	}
+
+	flag.Parse()
+
+	outFile, err := os.Create(resFile)
+	if err != nil {
+		quit(1, "Cannot open Output File\n")
+	}
+	defer func() {
+		err = outFile.Close()
+		if err != nil {
+			quit(1, "Error closeing Output File")
+		}
+	}()
+
+	fmt.Println("\nCommand Executor")
+	fmt.Println("--------------------------------------")
+	fmt.Println("Full Command:", command)
+	fmt.Println("Output File:", resFile)
+	fmt.Println("--------------------------------------")
+	status, msg := execCommand(command, outFile)
+	quit(status, msg)
+}
+
+func init() {
+	flag.StringVar(&command, "c", "", "Command to execute")
+	flag.StringVar(&resFile, "f", "resFile", "Otput File")
+}
+
+func printUsage() {
+	fmt.Println("\nCommand Executor")
+	fmt.Println()
+	fmt.Printf("Usage: %s [option] [command or filename]\n\n", os.Args[0])
+	fmt.Println("Options:")
+	flag.PrintDefaults()
+	quit(1, "")
+}
+
+func execCommand(command string, outFile *os.File) (status int, msg string) {
```

Comment:
`outFile *os.File` supposed to be represented as an interface, more precisely `io.Writer`